### PR TITLE
Increased memory limit

### PR DESCRIPTION
**Pull request by gignonje**

### CODE REVIEW

**Review by sempixel — APPROVED** [review]
oeuf curse!

File: conf.d/symfony.ini [hunk 1]
```diff
@@ -1 +1,2 @@
 date.timezone = Europe/Paris
+memory_limit = 4G
```
Comment (nicolasThal) [inline conf.d/symfony.ini]: i would rather put a different config file for the memory limit

**Review by nicolasThal — CHANGES_REQUESTED** [review]

File: conf.d/symfony.ini (hunk 1 above)
Comment (gignonje) [reply conf.d/symfony.ini]: Ok

**Comment by gignonje** [comment]
@nicolasThal Updated !

File: conf.d/memory.ini [hunk 2]
```diff
@@ -0,0 +1 @@
+memory_limit = 4G
```
Comment (kinok) [inline conf.d/memory.ini]: Didn't we agree on `2G`?

**Review by nicolasThal — APPROVED** [review]

File: conf.d/memory.ini (hunk 2 above)
Comment (gignonje) [reply conf.d/memory.ini]: Updated sorry

**Pull request merged**